src/muc_light/mod_muc_light_commands.erl

```diff
@@ -195,8 +196,8 @@ change_affiliation(Domain, RoomID, Sender, Recipient0, Affiliation) ->
 change_room_config(Domain, RoomID, RoomName, User, Subject) ->
     LServer = jid:nameprep(Domain),
     HostType = lserver_to_host_type(LServer),
-    MUCLightDomain = gen_mod:get_module_opt_subhost(
-                       Domain, mod_muc_light, mod_muc_light:default_host()),
+    HostType = mod_muc_light_utils:server_host_to_host_type(LServer),
```

Comment:
This looks very strange, it basically does the same as the line above and pattern-matches the results. After cleaning this up, please check if we even need `lserver_to_host_type` as this functions does the same as `mod_muc_light_utils:server_host_to_host_type`.

Reply:
yeah, was intended to be just one "HostType = "